requirements/dev.in

```diff
@@ -22,5 +22,6 @@ pytest-factoryboy
 pytest-lazy-fixture
 pytest-mock
 pytest-sugar
+responses
```

Comment:
I didn't realize that the project wasn't already using responses.

We should decide if we are actually planning to use responses more generally. Adding it to the project  trigger a single exception is probably overkill. One line of mocking could achieve the same thing.

Adding the one line of mocking is probably clearer anyways.

Let's just find out what Requests code is calling the Responses code and throw an exception in it.